changelogs/fragments/3964-scaleway_volume_add_region.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - scaleway_volume - All volumes are systematically created on par1 (https://github.com/ansible-collections/community.general/pull/3964).
```

```suggestion
  - scaleway_volume - all volumes are systematically created on par1 (https://github.com/ansible-collections/community.general/pull/3964).
```
This sounds like you implemented a feature that now all volumes are systematically created on par1. I somehow doubt that's what this PR does?

Also you are not mentioning the new `project` option.